source/fundamentals/connection/connect.txt

```diff
@@ -95,6 +95,13 @@ verify that the connection is successful:
 .. literalinclude:: /code-snippets/connection/stable-api-connect.js
    :language: javascript
 
+.. note::
+
+   As of {+driver-short+} v4.7, the driver automatically calls the ``MongoClient.connect()``
+   method when using the client to perform CRUD operations on your MongoDB deployment.
+   Call the ``MongoClient.connect()`` method explicitly to verify that the connection is
+   successful.
+
```

Comment:
@nbbeeken in the #node-driver-docs Slack channel posted proof that this automatic connection still works with bulk ops.

"Call the ``MongoClient.connect()`` method explicitly _if you would like to_ verify that the connection is successful" gets flagged by Vale for the italicized portion.

Reply:
How about: "...if you want to manually verify that the connection..."? As worded, it sounds like we're directing users to do it, which isn't necessary.